modules/kuberay-cluster/kuberay-autopilot-values.yaml

```diff
@@ -107,6 +107,7 @@ head:
     gke-gcsfuse/ephemeral-storage-limit: 20Gi
   nodeSelector:
     iam.gke.io/gke-metadata-server-enabled: "true"
+    cloud.google.com/compute-class: "Accelerator"
     cloud.google.com/gke-accelerator: "nvidia-tesla-t4"
```

Comment:
needs to be l4?

Reply:
On second thought, this is the head node and shouldn't be running an expensive GPU at all. The GPU worker group will be scaled up and down based on demand from scheduled Ray jobs.